### ISSUE TITLE
Project: Lore of Dark Souls

### ISSUE THREAD

**Issue by Cameron-Bigi** [newtfire/digitProjectDesign-Hub#56]
I'm a huge fan of creating and playing all different kinds of video games. One of my all time favorite game series are the Dark Souls trilogy published and developed by FromSoftware. But one of the game's few flaws is that the story is very hard to follow and understand. As you play throughout the entire game you'll have to piece together the story for your self and the way to do that is by reading every single item description in the game and talking to very single NPC. You have to do this because there are extremely important information in most of the hard to find items. I would like to create some kind of site that would help to organize all of this information into an easy and structured way so that players don't have to spend hours of there own time.

**Comment by trumerfe** [comment]
Hey Cameron! I'm a really big fan of the dark souls series and I'd be interested in participating in this project. I think having a single place where people can find all the lore information they need would be very helpful for people interested in learning about the story of the game. I'd recommend focusing in one of the three games for time/length/simplicity purposes, as the three games combined can get extremely complex in terms of raw information.

**Comment by Beeplosion** [comment]
Love the lore of the game and I agree, it's one of those franchises where it's up to the player to piece together the deep plot from numerous item entries and the land itself. Very fascinating a project to try and make it easier to follow all in one.
